src/constants/index.ts

```diff
@@ -1,8 +1,15 @@
 // sales params
-export const NFT_TOKEN_ADDRESS = '0xe330d543e9189450c36dc873aa3ab14106b1ee87'
-export const SALES_CONTRACT_ADDRESS = '0xfdd0d596350a78c3852a43d3b5910154b7c644db'
-export const CHAIN_ID = 11155111
-export const ETHERSCAN_URL = 'https://sepolia.etherscan.io'
+export const NFT_TOKEN_ADDRESS = '0xdeb398f41ccd290ee5114df7e498cf04fac916cb'
+export const SALES_CONTRACT_ADDRESS = '0xe65b75eb7c58ffc0bf0e671d64d0e1c6cd0d3e5b'
+export const CHAIN_ID = 137
+export const ETHERSCAN_URL = 'https://polygonscan.com'
+export const TRANSACTION_CONFIRMATIONS = 4
+
+// Params for testing
+// export const NFT_TOKEN_ADDRESS = '0xe330d543e9189450c36dc873aa3ab14106b1ee87'
+// export const SALES_CONTRACT_ADDRESS = '0xfdd0d596350a78c3852a43d3b5910154b7c644db'
+// export const CHAIN_ID = 11155111
+// export const ETHERSCAN_URL = 'https://sepolia.etherscan.io'
```

Comment:
These could be env values? Keep these values as defaults when env value is not supplied

Reply:
Done in commit 165a036